src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(2, 2))
+    .toEqual([1, 1]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(2, 1))
+    .toEqual([2]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(3, 2))
+    .toEqual([1, 2]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
-
+  expect(splitInteger(2, 3))
+    .toEqual([0, 1, 1]);
```

Comment:
The expected result of splitInteger(2, 3) should be [0, 1, 1], but the function is returning [0, 1, 1]. You may want to check your test case and make sure it's correct.